<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyboad_Dialog_class</class>
 <widget class="QDialog" name="Keyboad_Dialog_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>225</width>
     <height>264</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QPushButton" name="pB_Save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_key">
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lbl_frq">
      <property name="text">
       <string>[Hz]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Save Notes</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Buffer mode</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>base octave</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="sb_flats">
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="cb_sliding_mode">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>F7/f8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string># of sharps</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>F6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lbl_note">
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Decay mode</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_F4">
      <property name="text">
       <string>F4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string># of flats</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="sb_base_octave">
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="sb_sharps">
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>F2/F3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>slide mode</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>F5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="cB_buffer_mode"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Chord</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lbl_chord">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="sB_kbdbps"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
